person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative './student'
 require_relative './teacher'
+require_relative './nameable'
 
-class Person
+class Person < Nameable 
```

Comment:
- [ ] When I try to run your code, I get this error ``` superclass must be an instance of Class (given an instance of Module) (TypeError)``` so I suggest you make ```Nameable``` a class instead of a module. That should fix the error.